ViAn/Project/videoproject.cpp

```diff
@@ -111,7 +111,17 @@ std::map<ID, BasicAnalysis*> VideoProject::get_analyses() {
  */
 void VideoProject::read(const QJsonObject& json){
     m_tree_index = json["tree_index"].toString().toStdString();
-    this->m_video->read(json);
+    Video* vid = new Video();
+    vid->read(json);
+    if (vid->is_sequence()) {
+        // Need
```

Comment:
"Need" what?

Reply:
No idea.. Probably had more there before and only removed part of it